app/assets/stylesheets/main.scss

```diff
@@ -341,6 +341,14 @@ body{
   width: calc(100% - 25px);
 }
 
+#welcome-msg{
+  display:block;
+  padding: 15px;
+  color: white;
+  position: relative;
+  line-height: 20px;
+}
+
```

Comment:
Should probably have a fixed width and set ellipsis on long names, so it doesn't mess up the layout of other elements.